index.ts

```diff
@@ -72,7 +72,9 @@ export {
   getBmiStakingContract,
   get1InchStakingContract,
   getCommonOracleContract,
-  getIchiOracleAggregatorContract
+  getIchiOracleAggregatorContract,
+  getOneTokenFactoryContract,
+  getXICHIContract,
```

Comment:
Purely as a matter of convention, rename the xICHI.json to xIchi.json which will then renamed the output file to getXIchiContract which is more consistent with the camelCase naming we have so far.